<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-1</y>
      <width>611</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>581</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background-color: #f7f7ff;  /* پس‌زمینه تار با شفافیت */
    border: none;                             /* حذف حاشیه */
    border-radius: 15px;                      /* گرد کردن گوشه‌ها */
    color: black;                             /* رنگ متن آیتم‌ها */
    padding: 5px;                             /* فاصله داخلی */
    font-size: 14px;
}




QListWidget::item {
    padding: 8px;
}
QListWidget::item:selected {
    background-color: rgba(255, 255, 255, 50); /* رنگ آیتم انتخاب‌شده */
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>510</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <widget class="QPushButton" name="add1_pushButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(110, 166, 255);     
    border: none;               
    border-radius: 50px;          
    padding: 10px 20px;           
    font-size: 14px;              
}


QPushButton:hover {
    background-color: rgb(80, 140, 230);    
}

QPushButton:pressed {
    background-color: rgb(60, 120, 200);    
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../assets/icons/add.png</normaloff>../assets/icons/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="background_label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-30</y>
        <width>591</width>
        <height>681</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: #f7f7ff ;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>26</y>
        <width>531</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
    border: none;                             /* حذف حاشیه */
    border-radius: 20px;                      /* گرد کردن گوشه‌ها */
    color: black;                             /* رنگ متن داخل فیلد */
    font-size: 24px;

}</string>
      </property>
     </widget>
     <widget class="QLabel" name="reminder_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>511</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: #e6f2fe ;  /* رنگ  */
    border: none;                             /* حذف حاشیه */
    border-radius: 20px;                      /* گرد کردن گوشه‌ها */
    color: black;                             /* رنگ متن داخل فیلد */
    padding: 6px 10px;                        /* فاصله داخلی */
    font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="date_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>511</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: #e6f2fe ;  /* رنگ  */
    border: none;                             /* حذف حاشیه */
    border-radius: 20px;                      /* گرد کردن گوشه‌ها */
    color: black;                             /* رنگ متن داخل فیلد */
    padding: 6px 10px;                        /* فاصله داخلی */
    font-size: 14px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="getback_pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>560</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../assets/icons/arrow-back.ico</normaloff>../assets/icons/arrow-back.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="reminder_edit_pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>320</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(110, 166, 255);     
    border: none;               
    border-radius: 10px;           
	color:rgb(255, 255, 255);              
}


QPushButton:hover {
    background-color: rgb(80, 140, 230);    
}

QPushButton:pressed {
    background-color: rgb(60, 120, 200);    
}
</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="date_edit_pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>410</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(110, 166, 255);     
    border: none;               
    border-radius: 10px;           
	color:rgb(255, 255, 255);              
}


QPushButton:hover {
    background-color: rgb(80, 140, 230);    
}

QPushButton:pressed {
    background-color: rgb(60, 120, 200);    
}
</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>510</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(110, 166, 255);     
    border: none;               
    border-radius: 10px;           
	color:rgb(255, 255, 255);              
}


QPushButton:hover {
    background-color: rgb(80, 140, 230);    
}

QPushButton:pressed {
    background-color: rgb(60, 120, 200);    
}
</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
